### DISCUSSION TITLE
Unable to enable web login through CLI

### DISCUSSION THREAD

**Discussion by izuk** [photoprism/photoprism#4343] [Q&A]
#### 1. What is not working as documented?

Creating a user with the `--no-login` option seems impossible to undo.

#### 2. How can we reproduce it?

Create a new user with:

```
photoprism users add foo --no-login
```

There doesn't seem to be a way with the CLI to undo the option.  `mod` has no way to enable login.  `rm` removes the user, but a subsequent `add` just disables login again.

#### 3. What behavior do you expect?

There's a way to enable login through the CLI.

#### 4. What could be the cause of your problem?

Missing functionality or maybe a mistake on my part?

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: 240531-60b3a4628-Linux-AMD64-Plus

**Comment by lastzero** [comment]
=false

**Comment by MattTheBanana** [comment]
I'd like to add that this was not very intuitive, and I also would've logged a ticket if I didn't find this post. I think it should definitely say this in the docs or function as --no-login/-l BOOLEAN to be consistent.
